.github/workflows/ESLint.yml

```diff
@@ -0,0 +1,13 @@
+name: CI
```

Comment:
nejsem se jisty jestly to funguje, github ukaza nic. Ale pouzival jsem stejni config z [PR](https://github.com/cesko-digital/web/pull/44)

Reply:
diskusovali jsme z @HormCodes  ze se zepta nekoho kdo ma přístup k nastavení, a pak udela https://github.blog/2020-08-03-github-actions-improvements-for-fork-and-pull-request-workflows/. Tak zatim zkontolujeme jestly GithubActions funguji